src/main/resources/phases/0_plains.yml

```diff
@@ -1,5 +1,7 @@
 '0':
   name: Plains
+  # Material (Eg. GRASS_BLOCK) or Base64 Head Texture for Icon in "phases" command. Optional, Falls back to First Block
```

Comment:
Can you give some more info here on where the Base64 head texture would come from? Is there a good reference website you can point to? For example:

```
# Material (Eg. GRASS_BLOCK) or Base64 Head Texture for Icon in "phases" command. (Optional)
# If icon is not defined, then the first block of the phase will be used.
# See https://www.something.com for info on Base64 Head Textures
```